### DISCUSSION TITLE
"The debugger is not properly installed"

### DISCUSSION THREAD

**Discussion by Kayi-KuzinAlexandr25456** [CosmosOS/Cosmos#2887] [Q&A]
Every time I start, I see this: 
![Screenshot](https://github.com/CosmosOS/Cosmos/assets/118273508/fd2cd77a-d36a-4868-a94c-c0cff809b645)

Is it possible to make debugging work properly for me? My project configuration is vmware player (remote debug enabled).
I'm use vmware workstation 17 player, cosmos dev kit.

**Comment by 9xbt** [comment]
That doesn't seem like a debugger issue, it's more of a Cosmos issue I think. I would disable the Cosmos debugger and see if you still get the same error, though personally I never have needed to use the debugger, I just print messages in the OS

**Reply by Kayi-KuzinAlexandr25456** [reply]
The error still occurs.

**Reply by 9xbt** [reply]
just ignore it then
